### PR TITLE
reaction buttons and rebrand my-ocular addon

### LINKED ISSUE #2060: "Show my-ocular status" addon has outdated name and description
**Describe the bug**

The name and description of the `my-ocular` addon are outdated.

**Expected behavior**

The name should say "ocular status", not "my-ocular status", and the description should contain the new link for setting the status: https://ocular.jeffalo.net/dashboard

### PR DESCRIPTION
**Pull request by jeffalo**
**Resolves**

Resolves #2060 

**Changes**

adds reaction buttons for posts on the forums, reband addon to match new ocular branding

image

**Reason for changes**

**Tests**

yes, i tested on chrome on windows.

https://user-images.githubusercontent.com/40470736/114737672-8a180180-9d47-11eb-9ba0-8f87268653c1.mp4

### CODE REVIEW

**Comment by retronbv** [comment]
> yes, i tested on chrome os. 

Do you even have a chromebook?

**Comment by jeffalo** [comment]
> > yes, i tested on chrome os.
> 
> Do you even have a chromebook?

i do have a chromebook, but that line was a mistake. i should not watch youtube while writing a PR description...

**Comment by jeffalo** [comment]
before this gets merged, i need to fix a few bugs, like text color in dark mode.

on scratch
image
vs on ocular
image

**Comment by RedGuy12** [comment]
Are you going to add starring posts too?

**Comment by jeffalo** [comment]
> Are you going to add starring posts too?

that's something that will require a lot of work in terms of auth. i might come back to it in the future, however for now this works. i'll add a view on ocular button though.

File: addons/my-ocular/discuss.css
Comment (mxmou) [inline addons/my-ocular/discuss.css:2]: Use padding instead of margin so that there is no white space below left part of the post container.

File: addons/my-ocular/discuss.js
Comment (mxmou) [inline addons/my-ocular/discuss.js:10]: Please don't use `getComputedStyle` to do this: it doesn't update when scratchr2 or dark mode is enabled/disabled. Instead, include the default styles in the userstyle and override them in `scratchr2/forums.css` and `dark-www/experimental_forums.css`.

**Review by mxmou — CHANGES_REQUESTED** [review]

File: addons/my-ocular/discuss.css
Comment (jeffalo) [reply addons/my-ocular/discuss.css:2]: thanks

File: addons/my-ocular/discuss.js
Comment (jeffalo) [reply addons/my-ocular/discuss.js:10]: aw. okay then.

Comment (jeffalo) [reply addons/my-ocular/discuss.js]: fixed

**Comment by jeffalo** [comment]
it's ready. it still says maximouse's changes are pending but i changed them. did i miss something?

**Review by mxmou — APPROVED** [review]

**Comment by mxmou** [comment]
> it still says maximouse's changes are pending but i changed them. did i miss something?

Requests for changes are not automatically removed.

**Review by WorldLanguages — APPROVED** [review]
Will be merged soon for next release (DO NOT MERGE YET)

**Comment by jeffalo** [comment]
i forgot to change the text on the view on ocular button as we discussed

**Comment by jeffalo** [comment]
should we actually use the full text? on smaller screens this will start wrapping. i think the emoji + title text should be enough. or maybe if it just said "🔍 ocular"?

image

**Comment by WorldLanguages** [comment]
> if it just said "🔍 ocular"

do that

**Comment by return-misaPuding** [comment]
This addon doesn´t show up on the "Addons" page of ScratchAddons, the only addon showing up when I search "ocular" is "Show my-ocular status"

**Comment by mxmou** [comment]
@return-misaPuding It will be included in the next version (1.14.0).

**Pull request merged**
